dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.tds.searchbase/server.xml

```diff
@@ -0,0 +1,57 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="com.ibm.ws.wim.adapter.ldap.fat.tds.searchbase">
+	<featureManager>
+		<feature>appSecurity-2.0</feature>
+		<feature>ldapRegistry-3.0</feature>
+		<feature>servlet-3.1</feature>
+		<feature>securitylibertyinternals-1.0</feature>
+	</featureManager>
+
+	<!-- Test with the nested group filters and membershipAttribute for scope nested-->
+	<ldapRegistry id="LDAP" realm="TDSRealm" host="${ldap.server.4.name}" port="${ldap.server.4.port}" ignoreCase="true"
+		baseDN="o=ibm,c=us"
+		ldapType="IBM Tivoli Directory Server"
+		searchTimeout="8m">
+	  <ldapEntityType name="PersonAccount">
+        <objectClass>inetOrgPerson</objectClass>
+      </ldapEntityType>
+	  <ldapEntityType name="Group">
+        <objectClass>groupOfNames</objectClass>
+		<searchBase>ou=jgroups,o=ibm,c=us</searchBase> 
+      </ldapEntityType>
+      <loginProperty>uid</loginProperty>
+		<idsFilters
+			userFilter="(&amp;(uid=%v)(objectclass=ePerson))"
+			groupFilter="(&amp;(cn=%v)(|(objectclass=groupOfNames)(objectclass=groupOfUniqueNames)(objectclass=groupOfURLs)))"
+			userIdMap="*:uid"
+			groupIdMap="*:cn"
+			groupMemberIdMap="ibm-allGroups:member;ibm-allGroups:uniqueMember;groupOfNames:member;groupOfUniqueNames:uniqueMember" />
```

Comment:
Would probably be better if we used group properties?

Reply:
Overall, there is quite a bit of config in this file that seems unnecessary for the test. Would be more clear with concise configuration.